get_bullseye.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -uex
+
+bullseye_url="https://www.bullseye.com/download"
+dir_out="bullsye_dir.html"
+
+# Need to get a directory listing of files for download
+curl --location --retry 10 --retry-max-time 60 --silent --show-error \
+  "${bullseye_url}" -o $dir_out
+
+# Extract the filename from the listing
+dnl_fname=$(grep -i linux $dir_out | awk -F[\"] '{print $6}')
```

Comment:
You can replace `grep | awk` with:
```suggestion
dnl_fname=$(sed -ne '/Linux-x64/s/.*href="\(.*\)<\/a>.*/\1/p')
```